client/Components/Navigation/ServerStatus.jsx

```diff
@@ -5,69 +5,40 @@ import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { faCheckCircle, faTimesCircle, faBan } from '@fortawesome/free-solid-svg-icons';
 
 import './ServerStatus.scss';
+import NavbarItem from './NavbarItem';
 
 const ServerStatus = (props) => {
     const { connecting, connected, responseTime, serverType } = props;
     const { t } = useTranslation();
 
-    let className = '';
-    let toolTip = `${serverType} is`;
-    let pingText;
-    let icon = faCheckCircle;
+    const connectionStatus = t(
+        (connected && 'Connected') || (connecting && 'Connecting') || 'Disconnected'
+    );
 
-    if (connected) {
-        className += ' text-success';
-        toolTip += ' connected';
+    const connectionIcon = (connected && faCheckCircle) || (connecting && faTimesCircle) || faBan;
 
-        let pingClass;
+    const toolTip = `${serverType} is ${connectionStatus}`;
 
-        if (responseTime === undefined) {
-            pingText = <span>{t('Waiting for ping')}</span>;
-        } else {
-            if (responseTime < 150) {
-                pingClass = 'text-success';
-            } else if (responseTime < 300) {
-                pingClass = 'text-warning';
-            } else {
-                pingClass = 'text-danger';
-            }
+    const pingLevel = `ping-${
+        connected
+            ? responseTime
+                ? (responseTime < 150 && 'good') || (responseTime < 300 && 'mid') || 'bad'
```

Comment:
not sure why you've changed these from standard bootstrap classes

Reply:
The standard bootstrap colors looked a bit harsh compared to the mellow pink-ish of the rest of the theme, so I picked colors that signify the same but are a little more tame

Reply:
They're theme colours though, so should be the same as everywhere they're used - if we don't like them then we change them everywhere. I think they're ok though